source/tests/system/nidcpower_driver_api_tests.cpp

```diff
@@ -70,6 +70,19 @@ class NiDCPowerDriverApiTest : public ::testing::Test {
     ASSERT_EQ(kdcpowerDriverApiSuccess, response.status());
   }
 
+  void initiate()
+  {
+      dcpower::InitiateRequest request;
```

Comment:
[Minor] The spacing in the body of `initiate` and `fetch_multiple` is different than all the other methods. I think we're using two spaces per level of indent.

Reply:
Ah, I need to update my VS settings. Fixed the spacing.